.github/workflows/test.yml

```diff
@@ -63,20 +70,12 @@ jobs:
       - name: Start minikube
         uses: medyagh/setup-minikube@d8c0eb871f6f455542491d86a574477bd3894533 # 0.0.18
 
-      - name: Deploy cert-manager to minikube
-        run: |
-          helm repo add jetstack https://charts.jetstack.io --force-update
-          helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.14.5 --set installCRDs=true
-          sleep 5
-          kubectl wait --for=condition=Ready -n cert-manager --all pods
-
       - name: Deploy New Relic k8s-agents-operator to minikube
         run: |
           helm repo add k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
           helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
             --namespace=default \
             --set=licenseKey=${{ secrets.NEW_RELIC_LICENSE_KEY }} \
-            --set=newRelicHost=${{ vars.NEW_RELIC_HOST }} \
```

Comment:
Can we keep this line in place in case folks (like .NET) want to test locally and send data to Staging?

Reply:
How are you running github actions files locally? I could add that line to the makefile for local testing though.

Reply:
Oh, maybe that's my bad. I wasn't thinking GHA when I left the comment. Locally, we can't use the makefile either (silly .NET) - so I just set environment variables as needed and manually invoke the steps.

In any case, this is non-issue.